csharp/src/Ice/TraceLevels.cs

```diff
@@ -20,7 +20,7 @@ internal TraceLevels(Communicator communicator)
             TransportCategory = "Transport";
             ProtocolCategory = "Protocol";
             RetryCategory = "Retry";
-            LocationCategory = "Locator";
+            LocationCategory = "Locator"; // TODO why this inconsistency?
```

Comment:
I doubt there's a good reason for keeping this inconsistency!

Reply:
Replaced the symbol by LocatorCategory, as it's better to not change the existing category strings.